### ISSUE TITLE
Add deployed application link

### ISSUE THREAD

**Issue by Pericles001** [underscoDe/devflix-video-streaming-app#5]
Deploying the application is a great way to help people see its utility. We should deploy to help the project grow...
Heroku may be a good choice for this.

**Comment by kuroh123** [comment]
Do you mean to deploy on free heroku free hosting for now?

**Comment by Pericles001** [comment]
> Do you mean to deploy on free heroku free hosting for now?

Yes, exactly what I was thinking. Or any other free deployment service can help.

**Comment by kuroh123** [comment]
I can do it on heroku, but I will need your created heroku app so that you can manage the deployed app. Just create a new app and give me the link, i will deploy the app on that link.

**Comment by Pericles001** [comment]
Okay, let me see how to manage that. I will back soon.
